lib/l10n_utils/__init__.py

```diff
@@ -115,8 +118,9 @@ def render(request, template, context=None, ftl_files=None, activation_files=Non
             translations = set()
             for af in activation_files:
                 translations.update(ftl_active_locales(af))
-
             translations = sorted(translations)
+            if not translations:
```

Comment:
NIT: feels like a lot of churn for the `translations` variable in this section. Perhaps we should just set `translations = set()` back on line 116, remove line 118, and dedent this section?